packages/cli/README.md

```
@@ -180,7 +182,26 @@ $ mockoon-cli start --data cloud://def01727-aeb7-4cf1-9172-e0c38f22b224 --token
+When the admin API is enabled, bearer authentication is always enabled.
+
+You can provide the admin API token with `--admin-api-token` (or `MOCKOON_ADMIN_API_TOKEN`). If no token is provided, a secure token is auto-generated at startup and printed in the logs.
+
+- Provide one token to reuse it for all environments.
+- Provide one token per environment when starting multiple environments in one command.
+- Send the token with the `Authorization: Bearer <token>` header.
+
+By default, the admin API does not emit CORS headers, which prevents browser-based cross-origin requests. To allow specific origins (e.g. a frontend served from another domain), use `--admin-api-cors-origin` (or `MOCKOON_ADMIN_API_CORS_ORIGIN`):
+
+```bash
+$ mockoon-cli start --data ~/data.json --admin-api-cors-origin https://app.example.com
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)